linked-data-publishing/index.html

```diff
@@ -1722,13 +1722,13 @@ <h2>
     <p class="next">No solution mapping.</p>
     <p class="next">
       <em>At least, according to DBpedia.<br>
-          It turns out that <a href="http://www.imdb.com/name/nm2796745/">Adam DeVine</a> is actually an actor.</em>
+          It turns out that <a href="https://www.imdb.com/name/nm2796745/">Adam DeVine</a> is actually an actor.</em>
     </p>
   </div>
 
   <div class="slide" id="tpf-query-client">
     <h2>
-      Executing this query in the <a href="http://client.linkeddatafragments.org/#datasources=http%3A%2F%2Ffragments.dbpedia.org%2F2016-04%2Fen&query=SELECT%20%3Fperson%20%3Fcity%20WHERE%20%7B%0A%20%20%20%20%3Fperson%20rdf%3Atype%20dbpedia-owl%3AArtist.%0A%20%20%20%20%3Fperson%20dbpedia-owl%3AbirthPlace%20%3Fcity.%0A%20%20%20%20%3Fcity%20foaf%3Aname%20%22Waterloo%22%40en.%0A%7D">browser client</a><br>
+      Executing this query in the <a href="https://client.linkeddatafragments.org/#datasources=http%3A%2F%2Ffragments.dbpedia.org%2F2016-04%2Fen&query=SELECT%20%3Fperson%20%3Fcity%20WHERE%20%7B%0A%20%20%20%20%3Fperson%20rdf%3Atype%20dbpedia-owl%3AArtist.%0A%20%20%20%20%3Fperson%20dbpedia-owl%3AbirthPlace%20%3Fcity.%0A%20%20%20%20%3Fcity%20foaf%3Aname%20%22Waterloo%22%40en.%0A%7D">browser client</a><br>
```

Comment:
The data source URLs (inside of the query string) will need to be `https` as well, otherwise the browser won't be able to access them.